modules/text2vec-openai/clients/openai.go

```diff
@@ -130,18 +131,29 @@ func (v *client) Vectorize(ctx context.Context, input []string,
 	cfg moduletools.ClassConfig,
 ) (*modulecomponents.VectorizationResult, *modulecomponents.RateLimits, int, error) {
 	config := v.getVectorizationConfig(cfg, "document")
-	return v.vectorize(ctx, input, config.ModelString, config)
+	a, b, c, err := v.vectorize(ctx, input, config.ModelString, config)
```

Comment:
can we use more descriptive results instead of `a, b, c` we could use `res, rateLimits, totalTokens`?